engine/src/main/java/org/terasology/engine/i18n/TranslationSystemImpl.java

```diff
@@ -55,8 +53,8 @@ public void refresh() {
             Optional<Translation> asset = assetManager.getAsset(urn, Translation.class);
             if (asset.isPresent()) {
                 Translation trans = asset.get();
-                Uri uri = trans.getProjectUri();
-                if (uri.isValid()) {
+                ResourceUrn uri = trans.getProjectUri();
```

Comment:
```suggestion
                ResourceUrn urn = trans.getProjectUrn();
```

same for all occurrences